src/lib/device/journal_vdev.cpp

```diff
@@ -511,7 +547,11 @@ void JournalVirtualDev::Descriptor::truncate(off_t truncate_offset) {
         m_vdev.update_chunk_private(chunk, data);
 
         m_vdev.m_chunk_pool->enqueue(chunk);
-        HS_PERIODIC_LOG(TRACE, journalvdev, "adding chunk {} back to pool desc {}", chunk->chunk_id(), to_string());
+#if 0
```

Comment:
do we want to remove this code and put a comment here why we choose not to zero it after tuncate?